### PR TITLE
perf(asset-selector): do not render all assets at once

### LINKED ISSUE #3235: Memory Leak
Hey, I'm certain it is unintended for the wallet to consume ~2GB of RAM: 

image

How would I go about debugging the reason for this so I can report it?

### LINKED ISSUE #3380: QR will be not visible when creating multisig vault and using airgap account as a 1st signer
**Describe the bug**
QR will be not visible when creating multisig vault and using airgap account as a 1st signer 

**To Reproduce**
1. Import airgap acc
2. Click on create multisig 
3. add airgap acc as a 1st signer
4. add 2nd signer
5. Click on next 
6. see an issue

**Expected behavior**
qr code should be visible

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macos
 - Browser [ chrome,
 - reproducible in extension on commit [ffeabfe](https://github.com/superhero-com/superhero-wallet/commit/ffeabfe91937bdc68e64ad030165ef3b40b1b712)

**Additional context**
this issue is hard to reproduce because it's related to memory leaks issue and could be reproducible from time to time. 

https://github.com/user-attachments/assets/acb82163-bced-4bed-9434-00fbf0b2e77a

а

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #3235,
fixes #3380

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM.

File: src/popup/components/Modals/AssetSelector.vue [hunk 1]
```diff
@@ -24,19 +24,27 @@
...
-    >
-      <AssetListItem
-        v-for="asset in accountAssetsToDisplay"
-        :key="asset.contractId"
-        :asset="asset"
-        :selected="isAssetSelected(asset)"
-        prevent-navigation
-        @click="resolve(asset)"
-      />
+    <div ref="innerScrollElem">
```
Comment (peronczyk) [inline src/popup/components/Modals/AssetSelector.vue:27]: If this element wraps the infinite scroll it's not `inner` but `outer` :)

File: src/popup/components/Modals/AssetSelector.vue [hunk 2]
```diff
@@ -86,15 +100,26 @@ export default defineComponent({
...
     const { accountAssetsFiltered } = useAccountAssetsList({
       searchTerm,
       withBalanceOnly: props.withBalanceOnly,
     });
 
-    const accountAssetsToDisplay = computed(() => (props.protocol)
-      ? accountAssetsFiltered.value.filter(({ protocol }) => protocol === props.protocol)
-      : accountAssetsFiltered.value);
+    const appInnerElem = computed<HTMLElement | null | undefined>(
+      () => innerScrollElem.value?.parentElement?.parentElement,
```
Comment (peronczyk) [inline src/popup/components/Modals/AssetSelector.vue:114]: Which element exactly is accessed here?
To be honest using `parentElement` twice in the Vue environment looks like rather bad approach.

**Review by peronczyk — APPROVED** [review]

File: src/popup/components/Modals/AssetSelector.vue (hunk 1 above)
Comment (CedrikNikita) [reply src/popup/components/Modals/AssetSelector.vue:27]: This is a chosen class name wording in all similar situations, We can change it everywhere, if you think that it is more suitable.

**Comment by Liubov-crypto** [comment]
LGTM

**Pull request merged**